_rules/image-not-in-acc-tree-is-decorative-e88epe.md

```diff
@@ -0,0 +1,271 @@
+	ctx.font = '30px Arial'
+	ctx.fillText('ACT Rules!', 20, 40)
+</script>
+```
+
+### Inapplicable
+
+#### Inapplicable Example 1
+
+This `img` is [included in the accessibility tree][] because the `alt` attribute is not empty (`""`).
+
+```html
+<img src="/test-assets/shared/w3c-logo.png" alt="W3C" />
+```
+
+#### Inapplicable Example 2
+
+This `img` is neither [visible][] nor [included in the accessibility tree][].
+
+```html
+<img src="/test-assets/shared/w3c-logo.png" style="display:none" alt="" />
+```
+
+#### Inapplicable Example 3
+
+This `img` is not [visible][] because it is positioned off screen.
+
+```html
+<style>
+	img {
+		position: absolute;
+		top: -9999em;
+	}
+</style>
+<img src="/test-assets/shared/w3c-logo.png" alt="" />
+```
+
+#### Inapplicable Example 4
+
+This `svg` is ignored because it is a child of a link that has provides its [accessible name][].
+
+```html
+<a href="//example.org" aria-label="SVG star">
+	<svg height="200" xmlns="http://www.w3.org/2000/svg">
+		<polygon points="100,10 40,180 190,60 10,60 160,180" fill="yellow" />
+	</svg>
+</a>
+```
+
+#### Inapplicable Example 5
+
+```html
+<svg role="img" aria-label="Hello"></svg>
+```
+
+#### Inapplicable Example 6
+
+This `canvas` is not [visible][] because it is completely transparent.
+
+```html
+<canvas width="200" height="200"></canvas>
+```
+
+#### Inapplicable Example 7
+
+```html
+<canvas role="img" aria-label="Hello"></canvas>
+```
+
+#### Inapplicable Example 8
+
+This `img` is [included in the accessibility tree][].
+
+**Note**: While it might be better for the PDF icon to be ignored by assistive technologies, because assistive technologies will announce "PDF" twice, the image is not [purely decorative][]. Having assistive technologies ignore it is not required by [Success Criterion 1.1.1 Non-text content][].
+
+```html
+<img src="pdf.gif" alt="PDF" /> PDF document
+```
+
+[accessible name]: #accessible-name 'Definition of accessible name'
+[visible]: #visible 'Definition of Visible'
+[included in the accessibility tree]: #included-in-the-accessibility-tree 'Definition of Included in the accessibility tree'
+[semantic role]: #semantic-role 'Definition of Semantic Role'
+[explicit semantic role]: #explicit-role 'Definition of Explicit semantic role'
+[purely decorative]: https://www.w3.org/TR/WCAG21/#dfn-pure-decoration 'WCAG definition of Pure decoration'
+[success criterion 1.1.1 non-text content]: https://www.w3.org/TR/WCAG21/#non-text-content
+[conforming alternative version]: https://www.w3.org/TR/WCAG21/#dfn-conforming-alternate-version 'WCAG definition of Conforming alternative version'
+[flat tree]: https://drafts.csswg.org/css-scoping/#flat-tree 'CSS Scoping definition of Flat tree, working draft'
+[descendant]: https://dom.spec.whatwg.org/#concept-tree-descendant 'DOM definition of Descendant, 2020/03/06'
+[named from author]: https://www.w3.org/TR/wai-aria-1.1/#namecomputation 'WAI-ARIA definition of Named from author'
```

Comment:
Just to make sure that you really wanted to use that URL instead of this one:
https://www.w3.org/TR/wai-aria/#namecalculation